src/index.html

```diff
@@ -1,5 +1,7 @@
 <!doctype html>
-<html lang="en">
+<html
```

Comment:
The `<html>` tag is incorrectly self-closing. It should be a standard opening tag like `<html>` without the self-closing slash.